Sources/Models/Views/ActivityindicatorView.swift

```diff
@@ -35,6 +35,7 @@ public struct ActivityindicatorView: IBDecodable, ViewProtocol, IBIdentifiable {
     public let variations: [Variation]?
     public let backgroundColor: Color?
     public let tintColor: Color?
+    public let hidden: Bool?
```

Comment:
It demand more work but maybe rename it to isHidden everywhere
It demands mapping like here https://github.com/IBDecodable/IBDecodable/blob/master/Sources/Models/Views/Picker.swift#L47